docs/compute-systems/jupyter-and-ipython.md

```diff
@@ -110,7 +109,7 @@ jupyter kernelspec list
 ```
 
 To use a kernel, specify it by name when invoking a command. For
-example, to use the R 3.4.0 interpreter on Cheyenne in the Jupyter
+example, to use the R 3.4.0 interpreter on Casper in the Jupyter
```

Comment:
Use R 4.3 and r-4.3 here, since 3.4 is no longer available.

Reply:
Done and thanks!